components/BackButton.tsx

```diff
@@ -0,0 +1,19 @@
+import { FC } from 'react';
+import { LeftChevron } from './icons/sidebar_icons';
+import { useRouter } from 'next/router';
+
+const BackButton: FC = () => {
+  const router = useRouter();
+
+  return (
+    <div
```

Comment:
Make this a button so it's accessible (tabable, screen readers, etc.)
Also replace closing tag
```suggestion
    <button
```